zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -210,33 +210,56 @@ def channels2zerver_stream(slack_dir, realm_id, added_users):
 
 def channelmessage2zerver_message(slack_dir, channel, added_users, added_channels):
     json_names = os.listdir(slack_dir + '/' + channel)
+    users = json.load(open(slack_dir + '/users.json'))
     zerver_message = []
     msg_id_count = 1
+
+    #Sanitize the message text
+    def sanitize(string):
+        save_string = string
```

Comment:
`saved_string = str(string)` to be safe. I just got hit by this bug recently.

Reply:
E.g. https://github.com/zulip/zulip/pull/6841/files#diff-e263b61719994fe4bc93c9e43677ed5fR101